src/client/time/browser_clock.ts

```diff
@@ -0,0 +1,29 @@
+import { CancelTimer } from '../../shared/time/clock'
+import { Clock } from '../../shared/time/clock'
+
+function setTimeout(cb: (...args: any[]) => void, seconds: number): CancelTimer {
+  const handle = window.setTimeout(cb, seconds * 1e3)
```

Comment:
Not a huge fan of the repeated constant. Either make it a variable or implement conversion functions.